cmd/noms/noms_ds_test.go

```diff
@@ -25,21 +26,25 @@ type nomsDsTestSuite struct {
 
 func (s *nomsDsTestSuite) TestEmptyNomsDs() {
 	dir := s.DBDir
+	dbName := dir+"/name"
```

Comment:
Presumably `s.DBDir` is already an empty directory, no?

Can't you just use that, which should already exist? If you change it, and the test fails then feel free to keep it like this -- I'm not going to ask you to debug that kind of thing :-)